packages/ai/tests/ai/web/test_server.py

```
@@ -0,0 +1,140 @@
+            'task',
+            'task_http',
+        }
+        assert ALLOWED_MODULES == expected
+
+
+# ============================================================================
+# WebServer.use() tests
+# ============================================================================
+
+
+def _make_server():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_make_server`

(ANN202)